Makefile

```diff
@@ -77,6 +77,8 @@ run-collector: clone-tnf-secrets stop-running-collector-container
 		-e SERVER_ADDR=':${HOST_PORT}' \
 		-e SERVER_READ_TIMEOUT=10 \
 		-e SERVER_WRITE_TIMEOUT=10 \
+		-e AWS_ACCESS_KEY=$(shell jq -r ".CollectorAWSAccessKey" "./tnf-secrets/collector-secrets.json") \
```

Comment:
What is a meaning of `shell` here? I think it would work without it.

Reply:
Sometimes in Makefile's you have to specify.